### DISCUSSION TITLE
is it possible to change http status code for class-validator errors?

### DISCUSSION THREAD

**Discussion by Vakintoshka** [MichalLytek/type-graphql#832] [Q&A]
Hey guys, I do typegraph with apollo and currently it returns me http status 200 on invalid request.
Example - I do @isEmail validation and send value which is not email. Http status 200 and error is in the body. Is it any way how to change it to the 400 code?
Thanks

**Comment by MichalLytek** [comment]
TypeGraphQL create a GraphQL schema, not a HTTP server, so it depends on server you use (mercurius, apollo, express-graphql). Go to their docs to find out how to format errors or send different http codes.

TypeGraphQL throws `ArgumentValidationError` so you can import it to do `instance of` check.

**Reply by Vakintoshka** [reply]
Thank you for a quick response, and sorry that I've missed to share that I'm using Apollo. That way it's probably better to post this question in their discussions. Thanks!
